fecgov/openFEC
# [not merge]Fix candidate has committee without financial form showing blank financial summery

## Summary (required)
This PR will fix candidates that have a committee(have the record in **linkage** table) and not file any financial form, showing up the blank financial summary pages.

- Resolves [https://github.com/fecgov/fec-cms/issues/3336](https://github.com/fecgov/fec-cms/issues/3336)

_Include a summary of proposed changes._
1) remove the `LEFT` join for ofec_committee_totals_per_cycle_vw in ofec_candidate_totals_detail_mv
2) Add test case for the scenario: candidate have committee and without financial form 
in integration/test_candidate_committee_totals.py 


## How to test the changes locally
**Part 1:** openFEC side: check new V0172__remove_left_join_ofec_candidate_totals_detail_mv.sql
1) checkout branch
2) run **flyway migrate** or **invoke create_sample_db**
3) pytest
 


**Part 2:** need test from local cms side:
1) setup local api, point dev db
2) modify models/totals.py two class files point to tmp mv file
from **ofec_candidate_totals_detail_mv** change to **ofec_candidate_totals_detail_mv_tmp_jl**

class CandidateCommitteeTotalsPresidentia
class CandidateCommitteeTotalsHouseSenate
__tablename__ = 'ofec_candidate_totals_detail_mv_tmp_jl'

3) pytest
4) ./manage.py runsever
5) check endpoint: candidate/{candidate_id}/totals/   under tag: candidate
make sure it works well

6) setup local cms point to local api

7) Test Example
1) H2PA04192
prod: https://www.fec.gov/data/candidate/H2PA04192/
local: http://127.0.0.1:8000/data/candidate/H2PA04192/


2) S2MO00262
prod: https://www.fec.gov/data/candidate/S2MO00262/
local: http://127.0.0.1:8000/data/candidate/S2MO00262/


3) P00012799
prod: https://www.fec.gov/data/candidate/P00012799/
local: http://127.0.0.1:8000/data/candidate/P00012799/


## Impacted areas of the application
candidate profile page
